HW5/HW5 SQL.sql

```diff
@@ -0,0 +1,126 @@
+CREATE OR REPLACE TABLE temp_for_rolling 
+SELECT 
+	g.game_id AS game_id,
+	tbc.team_id AS team_id,
```

Comment:
I think we don't need to give alias names for game_id and team_id. They would come out just like that.

Reply:
Thanks!